app/views/support_interface/settings/cycles.html.erb

```diff
@@ -25,7 +25,7 @@
 <%= render SummaryListComponent.new(rows: {
   'Previous cycle year' => RecruitmentCycle.previous_year,
   'Current cycle year' => RecruitmentCycle.current_year,
-  'Next cycle year' => EndOfCycleTimetable.next_cycle_year,
```

Comment:
If we're going with `RecruitmentCycle`, does this mean we can lose this method off the `Timetable`, or are there other callers?

Reply:
Sorry forgot to remove, have done this now